src/test/java/com/foundation/salesforce/steps/ContactAcceptanceSteps.java

```diff
@@ -0,0 +1,268 @@
+/*
+ * @(#) ContactAcceptanceSteps.java Copyright (c) 2019 Jala Foundation.
+ * 2643 Av. Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ * Jala Foundation, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+package com.foundation.salesforce.steps;
+
+import com.foundation.salesforce.core.api.ContactApi;
+import com.foundation.salesforce.core.restClient.RestClientApi;
+import com.foundation.salesforce.core.utils.EndPoints;
+import com.foundation.salesforce.entities.Contact;
+import cucumber.api.java.en.Given;
```

Comment:
Please, place separator blank lines between groups of libraries.

Reply:
Done.